src/main/com/yetanalytics/markdoc.clj

```diff
@@ -72,9 +72,9 @@
   "Add `content` to the HTML doc template."
   [content template-file & {:keys [template-vars]}]
   (selm-parser/render (get-template template-file)
-                      (merge {:content content
-                              :sha @git-sha}
-                             template-vars)))
+                      (merge template-vars
+                             {:content content
+                              :sha @git-sha})))
```

Comment:
Cool - might want to include that in the documentation

Reply:
The documentation already says don't use this key (now, ofc, because it will be discarded). I think that's enough. Also if you think about it they will need to willingly reference the var IN the template alongside existing reference to {{content}} they also wrote themselves so this was an incredibly unlikely mistake to start.